Multiple matchbox-stroke instances would occupy whole screen and cause end user
confusing. Making matchbox-stroke singleton is one work around.

This patch allow only one instance of matchbox-stroke based on some mechanism
in matchbox-desktop.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [configuration]

Index: matchbox-stroke/matchbox-stroke.desktop
===================================================================
--- matchbox-stroke.orig/matchbox-stroke.desktop	2010-07-10 10:23:26.000000000 +0800
+++ matchbox-stroke/matchbox-stroke.desktop	2010-12-22 20:59:26.000000000 +0800
@@ -6,3 +6,5 @@
 Icon=matchbox-stroke.png
 Categories=Panel;Utility;MB
 X-MB-INPUT-MECHANSIM=True
+X-MB-SingleInstance=true
+StartupNotify=true
